diagnostic.md

```diff
@@ -3,9 +3,8 @@
 Using `inspect element`, write four selectors found on the main page for this
 GitHub repo.
 
-<!-- your answer starts here -->
+<div id="js-pjax-loader-bar" class="pjax-loader-bar">
```

Comment:
This is just an element declaration.  We are looking for four selectors.  A few examples are `body`, `div.pjax-loader-bar`, and `#js-pjax-loader-bar`.